.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "themes/drone-theme"]
+	path = themes/drone-theme
+	url = git@github.com:unicef/inventory-hugo-theme.git
```

Comment:
The git submodule still needs to be added for it to appear as a new change inside of this Pull Request. 

```sh
git submodule add --force --name inventory git@github.com:unicef/inventory-hugo-theme.git themes/inventory
```

After running this command, you should notice a `git clone` happen, as the git submodule is cloned into `themes/inventory` in the repo. You should see files from the upstream repo at this path.

Make sure to clean up the `.gitmodules` file after running this command, so only the single correct submodule appears. 🧹 

BTW, this was a [helpful resource on submodules](https://www.atlassian.com/git/tutorials/git-submodule) that helps me whenever I have to remember how to work with them! 📝

Reply:
@Idadelveloper It looks like you committed the files manually instead of adding the theme as a submodule. I'm going to clean it up this time since it is a fresh start, but note that manually committing the files does not link them to the upstream git repository via a submodule, and creates more work on the maintenance side.